cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -241,7 +241,11 @@ protected VaultConfig readVaultConfig(final Session<?> session) throws Backgroun
         }
     }
 
-    private VaultConfig readVaultConfig(final String token) {
+    protected String readVaultConfig(final Session<?> session) throws BackgroundException {
```

Comment:
Is it required to have this overridable by subclasses. Otherwise I would omit the single line method as it only has a single usage.